### ISSUE TITLE
Documentation: Installing Tekton Operator - ``` backticks blighting rendered markdown fenced code block

### ISSUE THREAD

**Issue by arnoldmashava** [tektoncd/website#596]
In the [Installing Tekton Operator](https://tekton.dev/docs/operator/install/#installing-tekton-operator) Section pertaining to the `Documentation for Operator-v0.70.x`, the final illustrative code block for the installation of the `Tekton Operator` pipelines, triggers, chains and dashboard could have been rendered with a repetition of the bottom ``` backticks that were intended for encapsulating the `kubectl apply -f` command into a fenced code block after rendering:

```bash
>_ kubectl apply -f https://raw.githubusercontent.com/tektoncd/operator/main/config/crs/kubernetes/config/all/operator_v1alpha1_config_cr.yaml
```
Not that there is anything vulgar with a little overdose of `*.md markdown` syntax, the minor glitch in the presentation of the code is comprehensible but the Documentation aesthetics and resulting visual stimulus could be easier on the *Wandering Eye* of the enchanted but also finicky novice...

**Comment by AlanGreene** [comment]
Thanks for reporting this @arnoldmashava. The source of that doc page is at https://github.com/tektoncd/operator/blob/main/docs/install.md

Feel free to open a PR removing the unnecessary backticks.

**Comment by arnoldmashava** [comment]
Thanks for the corrective insights @AlanGreene - yes, the `Edit this page` feature in the [Tekton Documentation](https://tekton.dev/docs/) automates the direct submission of proposed edits in the flagged Documentation pages to the Code Maintainers, but like all alluring and salacious things in life, there will always be the existence of shortcomings in the applicability of this vaunted `Pull Request` methodology.

This is exemplified by the Documentation for [Tekton Catalog Tasks](https://github.com/tektoncd/catalog/task), where in the Task for [Building and Uploading of Container Images using Kaniko](https://artifacthub.io/packages/tekton-task/tekton-catalog-tasks/kaniko), the wrong versioning date that was assigned to Release `Ver. 0.6.x`

![Screenshot_20240312_195534](https://github.com/tektoncd/website/assets/5759101/4ec298c1-a63d-4bf1-a30a-f96f2f3b392e)

is not editable through the existing source `README.md` file that corresponds to the `/tektoncd/catalog/tree/task/kaniko/0.6` [source repository](https://github.com/tektoncd/catalog/blob/main/task/kaniko/0.6/README.md).

In conclusion, perhaps I could as well deal with this versioning minor glitch at some later passage of time as a separately filed [Issue](https://github.com/tektoncd/website/issues) but for now, we can work on closing Issue #596 after the approval of Pull Request [2047](https://github.com/tektoncd/operator/pull/2047).

Muchas Gracias y Saludos Cordiales,

Arnold Mashava.

**Comment by AlanGreene** [comment]
The content on Artifact Hub is not related to the Tekton Website and I'm not sure I understand the point you're trying to make about pull requests in this case.

If there's an issue with the catalog content you could open an issue in the Tekton Catalog repo at https://github.com/tektoncd/catalog/issues for the attention of the maintainers. I'm not sure what the source of that date is, it's entirely possible it could be something on the Artifact Hub side.
